packages/eslint-plugin-pf-codemods/src/rules/v6/chartsImportMoved/chartsImport-moved.test.ts

```diff
@@ -0,0 +1,106 @@
+  InvalidTests,
+  createValidTest,
+  createInvalidTest,
+} from "../../helpers/testHelpers";
+
+const specifiersToMove = [
+  'Chart',
+  'ChartArea',
+  'ChartAxis',
+  'ChartBar',
+  'ChartBoxPlot',
+  'ChartBullet',
+  'ChartBulletComparativeErrorMeasure',
+  'ChartBulletComparativeMeasure',
+  'ChartBulletComparativeWarningMeasure',
+  'ChartBulletPrimaryDotMeasure',
+  'ChartBulletPrimarySegmentedMeasure',
+  'ChartBulletQualitativeRange',
+  'ChartContainer',
+  'ChartCursorContainer',
+  'ChartCursorTooltip',
+  'ChartCursorFlyout',
+  'ChartDonut',
+  'ChartDonutThreshold',
+  'ChartDonutUtilization',
+  'ChartGroup',
+  'ChartLabel',
+  'ChartLegend',
+  'ChartLegendTooltip',
+  'ChartLegendTooltipContent',
+  'ChartLegendTooltipLabel',
+  'ChartLine',
+  'ChartPie',
+  'ChartPoint',
+  'ChartScatter',
+  'ChartStack',
+  'ChartTheme',
+  'ChartThemeColor',
+  'ChartThreshold',
+  'ChartTooltip',
+  'ChartVoronoiContainer',
+  'createContainer',
+  'getInteractiveLegendEvents',
+  'getInteractiveLegendItemStyles',
+  'getCustomTheme',
+  'getTheme',
+  'getThemeColors'
+];
+
+const validTests: ValidTests = [];
+const invalidTests: InvalidTests = [];
+
+specifiersToMove.forEach((specifier) => {
+  validTests.push(createValidTest(`<${specifier} />`));
+  validTests.push(
+    createValidTest(
+      `import { ${specifier} } from '@patternfly/react-charts/victory';`
+    )
+  );
+
+  const errorMessage = `${specifier} has been moved. This rule will update import paths.`;
+  invalidTests.push(
+    createInvalidTest(
+      `import { ${specifier} } from '@patternfly/react-charts';`,
+      `import {\n\t${specifier}\n} from '@patternfly/react-charts/victory';`,
+      [{ message: errorMessage, type: "ImportDeclaration" }]
+    )
+  );
+  invalidTests.push(
+    createInvalidTest(
+      `import { ${specifier} as CustomSpecifier } from '@patternfly/react-charts';`,
+      `import {\n\t${specifier} as CustomSpecifier\n} from '@patternfly/react-charts/victory';`,
+      [{ message: errorMessage, type: "ImportDeclaration" }]
+    )
+  );
+  invalidTests.push(
+    createInvalidTest(
+      `import { ${specifier} } from '@patternfly/react-charts/dist/esm/components/index.js';`,
+      `import {\n\t${specifier}\n} from '@patternfly/react-charts/dist/esm/victory/components/index.js';`,
+      [{ message: errorMessage, type: "ImportDeclaration" }]
+    )
+  );
+  invalidTests.push(
+    createInvalidTest(
+      `import { ${specifier} } from '@patternfly/react-charts/dist/js/components/index.js';`,
+      `import {\n\t${specifier}\n} from '@patternfly/react-charts/victory';`,
+      [{ message: errorMessage, type: "ImportDeclaration" }]
+    )
+  );
+  invalidTests.push(
+    createInvalidTest(
+      `import { ${specifier} } from '@patternfly/react-charts/dist/dynamic/components/index.js';`,
+      `import {\n\t${specifier}\n} from '@patternfly/react-charts/victory';`,
+      [{ message: errorMessage, type: "ImportDeclaration" }]
+    )
```

Comment:
Is it intended that for the cjs and dynamic imports it changes it to just `/victory`, but it keeps the esm import as explicitly esm?

Reply:
I copied the Modal tests here. I'll update it